app/src/main/java/com/cornellappdev/volume/navigation/NavigationItem.kt

```diff
@@ -26,15 +25,15 @@ sealed class NavigationItem(
 
     object Magazines : NavigationItem(
         Routes.MAGAZINES.route,
-        R.drawable.ic_magazines_gray,
-        R.drawable.ic_magazines_orange,
+        R.drawable.ic_magazine_icon_sel,
```

Comment:
Spell out sel -> selected and un -> unselected so things are more clear/explicit